vampyr/functions/Polynomial.cpp

```diff
@@ -0,0 +1,25 @@
+#include "pybind11/pybind11.h"
+
+#include "PyRepresentableFunction.h"
+
+#include "functions/Polynomial.h"
+
+using namespace mrcpp;
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace vampyr {
+
+void polynomial(py::module &m, py::class_<RepresentableFunction<1>, PyRepresentableFunction<1>> &repfunc1d) {
+
+
+py::class_<Polynomial>(m, "Polynomial", repfunc1d)
```

Comment:
Add a comment on the `*`

Reply:
Don't know what happen, somehow I missed the comments I had previous added. Will do it again  for all the *